Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.py

```diff
@@ -1775,6 +1874,210 @@ def get_last_alert_fetched_time(last_run, alert_time_to_fetch):
     return last_alert_fetched_time
 
 
+def list_indicators_command(client: MsClient, args: Dict[str, str]) -> Tuple[str, Optional[Dict], Optional[List]]:
+    """
+
+    Args:
+        client: MsClient
+        args: arguments from CortexSOAR. May include 'indicator_id'
+
+    Returns:
+        human_readable, outputs.
+    """
+    indicators = client.list_indicators(args.get('indicator_id'))
+    limit = int(args.get('limit', 50))
+
+    indicators = indicators[:limit]
```

Comment:
cant we send the limit in the api request?

Reply:
nope